webpages/about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>

```

Comment:
I think you could remove the `webpages` folder and it's files, because they're also in `index/templates`.

Reply:
Done